python-checks/src/main/java/org/sonar/python/checks/CheckList.java

```diff
@@ -150,6 +151,7 @@ private CheckList() {
       FunctionReturnTypeCheck.class,
       FunctionUsingLoopVariableCheck.class,
       GenericExceptionRaisedCheck.class,
+      GroupReplacementCheck.class,
```

Comment:
I think it should go after `GraphemeClustersInClassesCheck` to be alphabetically sorted